src/app/app.module.ts

```diff
@@ -1,6 +1,6 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
-import { DeviceDetectorModule } from 'ngx-device-detector';
+// import { DeviceDetectorService } from 'ngx-device-detector';
```

Comment:
Unclear to me why this is present - it doesn't seem to be used anywhere in the app 🤔 
We can remove it from the `package.json` file I think 👍

Reply:
Yes remove from `package.json`  file and remove from all the place (don't comment it).

Reply:
✅ removed 👍